sites/default/sqlconf.php

```diff
@@ -1,30 +1,30 @@
 <?php
 //  LibreEHR
 //  MySQL Config
-//  Referenced from /library/sqlconf.php.
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'libreehr';
-$pass	= 'libreehr';
+$pass	= 'libreehr1';
```

Comment:
This file should not be touched. Please use gitignore.

Reply:
What he said...